src/components/ChooseCustomer.js

```diff
@@ -0,0 +1,70 @@
+import React, { useRef, useEffect, useState } from 'react';
+import axios from 'axios';
+import {withRouter, Redirect} from 'react-router-dom'
+import { Button, Card, Form, List } from 'antd';
+import 'antd/dist/antd.css';
+import { useStoreState, useStoreActions } from 'easy-peasy';
+
+
+
+const ChooseCustomer = ({ props }) => {
```

Comment:
Remove `{ props }` from here. If you wanted to pass props, then it would be `... = (props) => ...`

Reply:
removed